subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -24,6 +24,14 @@
 
 static radio_isr_fp sfp_radio_isr;
 
+#if defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER)
+static u32_t last_pdu_end_us;
+static inline void increment_last_pkt_end_time(void)
```

Comment:
this seems a bit long and doesn't match the variable name, how about `inc_last_pdu_end()`?

Reply:
OK!